src/winstanley/WdlParserDefinition.scala

```diff
@@ -45,8 +45,8 @@ class WdlParserDefinition extends ParserDefinition {
 
   def createFile(viewProvider: FileViewProvider): PsiFile = new WdlFile(viewProvider)
 
-  def spaceExistanceTypeBetweenTokens(left: ASTNode, right: ASTNode): SpaceRequirements = SpaceRequirements.MAY
+  override def spaceExistenceTypeBetweenTokens(left: ASTNode, right: ASTNode): SpaceRequirements = SpaceRequirements.MAY
```

Comment:
I assume this override is to make sure we really are overriding something (because it looks like there was a typo previously?

Maybe it's sensible to mark `override` on the other properties too?

Reply:
It was a typo by JetBrains; we were overriding before but for some reason got away without the `override` keyword